src/Services/SettingsBase.cs

```diff
@@ -31,6 +37,16 @@ protected SettingsBase(IFolderData folder, IAsyncSerializer<Stream> settingSeria
             Folder = folder;
         }
 
+        /// <summary>
+        /// Gets or sets the property that determines whether to flush default values to disk.
+        /// </summary>
+        protected bool FlushDefaultValues { get; set; } = true;
+
+        /// <summary>
+        /// Gets or sets the property that determines whether to only persist settings when they are changed through <see cref="SetSetting{T}"/>.
+        /// </summary>
+        protected bool PersistIfChanged { get; set; }
```

Comment:
Adjusting this to align with `FlushDefaultValues` naming

```suggestion
        protected bool FlushOnlyChangedValues { get; set; }
```